laterpay/__init__.py

```diff
@@ -358,6 +359,13 @@ def get_access_params(self, article_ids, lptoken=None):
             'article_id': article_ids,
         }
 
+        if muid:
+            # TODO: The behavior when lptoken and muid are given is not yet
```

Comment:
Thanks for you input here! The benefit that would come with doing some validation here are purchase dialog URLs that are working and not incorrect due to invalid arguments.